### PR TITLE
[WIP] Send ETH funds

### PR DESCRIPTION
**Pull request by ozalexo**
This change is [Reviewable](https://reviewable.io/reviews/ozalexo/chronomint-rn/41)

### CODE REVIEW

**Review by molochkoego — APPROVED** [review]

File: packages/ethereum/redux/thunks.js [hunk 1]
```diff
@@ -52,4 +53,149 @@ export const updateEthereumBalance = ({ tokenSymbol, address, balance, amount })
...
+// export const updateEthereumTxDraftChainId = ({ chainId, masterWalletAddress }) => (dispatch) => {
+//   return new Promise((resolve, reject) => {
+//     try {
+//       dispatch(Actions.updateEthereumTxDraftChainId({ chainId, masterWalletAddress }))
+//       return resolve()
+//     } catch (e) {
+//       return reject(e)
+//     }
+//   })
+// }
```
Comment (molochkoego) [inline packages/ethereum/redux/thunks.js]: There were my comments. Should we remove them?

File: src/images/index.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ export sending_50_circle_big from './sending-50-circle-big.png'
 export sending_50_circle_small from './sending-50-circle-small.png'
 export sending_75_circle_big from './sending-75-circle-big.png'
 export sending_75_circle_small from './sending-75-circle-small.png'
-export spinner_wheel_svg from './spinner-wheel.svg'
+// export spinner_wheel_svg from './spinner-wheel.svg'
```
Comment (molochkoego) [inline src/images/index.js:100]: Should we uncomment this line?

File: src/screens/Wallet/Send/Modals/ConfirmSendModal/ConfirmSendModalContainer.js [hunk 3]
```diff
@@ -10,16 +10,19 @@ import { bindActionCreators } from 'redux'
...
 
 const mapStateToProps = (state) => {
+  const masterWalletAddress = getCurrentWallet(state)
+
   return {
     prices: {
       BTC: {
         USD: 1499,
       },
     },
```
Comment (molochkoego) [inline src/screens/Wallet/Send/Modals/ConfirmSendModal/ConfirmSendModalContainer.js]: I could delete this and use prices selector from market

**Review by molochkoego — APPROVED** [review]

File: src/screens/Wallet/Send/Modals/ConfirmSendModal/ConfirmSendModalContainer.js (hunk 3 above)
Comment (ozalexo) [reply src/screens/Wallet/Send/Modals/ConfirmSendModal/ConfirmSendModalContainer.js]: It will be great.

File: src/images/index.js (hunk 2 above)
Comment (ozalexo) [reply src/images/index.js:100]: Nope. We can't import svg files directly. Usage example may be found here: https://github.com/vault-development/react-native-svg-uri

File: packages/ethereum/redux/thunks.js (hunk 1 above)
Comment (ozalexo) [reply packages/ethereum/redux/thunks.js]: Yes, the may be removed safely.

**Pull request merged**